libraries/dm/include/dfuse/dm/log_appender.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <fc/log/appender.hpp>
+
+namespace dfuse { namespace dm  {
+
+   class log_appender : public fc::appender
+   {
+       public:
+            log_appender( const fc::variant& args );
+            log_appender();
+
+            virtual ~log_appender();
+            void initialize( boost::asio::io_service& io_service ) {
+                 setbuf(stdout, NULL);
```

Comment:
I don't think you should hijack `stdout` like this. Can you use a file instead?

Reply:
While I agree the `stdout` is not always a good thing to hijack, in this specific case, someone is expected to have `deep-mind = true` which is opt-in.

All the underlying reading tools are currently built around reading `stdout`. The `stdout` was a good choice because by default, `nodeos` outputs to `stderr`. So it was making sense for us.

Using a file based is definitely not something we want to use, we don't want to deal with its constraints (need to truncate it, disk access, disk full, etc).

I know there was some discussion about using a `fifo` instead, which would be a good compromise. But it seemed to be ok for now to still push through `stdout` for the initial version.

Reply:
Can you just add a `fflush` call to `out`?

Reply:
Discussion now happening on https://github.com/EOSIO/fc/pull/148